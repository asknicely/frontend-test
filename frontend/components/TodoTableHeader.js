import React from 'react';

function TodoTableHeader() {
  return (
    <thead>
      <tr>
        <th>#</th>
        <th>User</th>
        <th>Description</th>
        <th></th>
      </tr>
    </thead>
  );
}

export default TodoTableHeader;
