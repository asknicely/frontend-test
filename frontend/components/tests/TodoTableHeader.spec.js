import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import TodoTableHeader from '../TodoTableHeader';

describe('TodoTableHeader', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<TodoTableHeader />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render TodoTableHeader correctly', () => {
    const wrapper = shallow(<TodoTableHeader />);

    expect(wrapper).toMatchSnapshot();
  });
});
