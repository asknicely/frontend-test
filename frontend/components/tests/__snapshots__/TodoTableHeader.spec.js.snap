// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoTableHeader should render TodoTableHeader correctly 1`] = `ShallowWrapper {}`;

exports[`TodoTableHeader snapshot renders 1`] = `
<thead>
  <tr>
    <th>
      #
    </th>
    <th>
      User
    </th>
    <th>
      Description
    </th>
    <th />
  </tr>
</thead>
`;
